### DISCUSSION TITLE
Rutile Ore for obtaining titanium

### DISCUSSION THREAD

**Discussion by Astropye** [TechReborn/TechReborn#2881] [Ideas]
There isn't really any good way of obtaining titanium yet. 2 small piles of titanium dust from bauxite ore aren't great for larger projects (right now I'm just trying to upgrade my blast furnace so I can smelt tungsten) and Heart of the Seas are too rare of an item to be viable either. This site seems to have a fairly thorough guide on the processing of the ore into Titanium: https://www.chemguide.co.uk/inorganic/extraction/titanium.html
Seems you'd need heat, chlorine, and some form of carbon for the first stage, and magnesium or sodium for the second. There are other ores irl that are processed into titanium that might work better in the mod, rutile was just the first one that showed up.

**Comment by Ayutac** [comment]
Sounds great, and I am also glad that HotS does not produce enough to be viable for mass production. My balancing was right :D

**Comment by Ayutac** [comment]
If we put the intermediate form into a cell, we only need textures for
- ore block (probably end?)
- dust
- small dust
- intermediate fluid

**Comment by Ayutac** [comment]
Have you considered just using uu matter for the titanium?
